lib/src/StaticFileRouter.h

```diff
@@ -93,6 +93,10 @@ class StaticFileRouter
     {
         return implicitPage_;
     }
+    void setDefaultHandler(DefaultHandler handler)
```

Comment:
Here should be `DefaultHandler &&handler`, because this method always is called with an r-reference parameter.